src/main/java/com/mraof/minestuck/computer/DiskBurner.java

```diff
@@ -0,0 +1,50 @@
+package com.mraof.minestuck.computer;
+
+import com.mraof.minestuck.blockentity.ComputerBlockEntity;
+import com.mraof.minestuck.network.MSPacketHandler;
+import com.mraof.minestuck.network.computer.*;
+import com.mraof.minestuck.util.MSTags;
+
+import java.util.ArrayList;
+
+public class DiskBurner extends ButtonListProgram
+{
+	public static final String NAME = "minestuck.disk_burner_program";
+	public static final String BURN_SERVER_DISK = "minestuck.burn_server_disk";
+	public static final String BURN_CLIENT_DISK = "minestuck.burn_client_disk";
+	public static final String CHOOSE = "minestuck.choose";
```

Comment:
Perhaps add to the unlocalized name slightly to make it specific to the disk burner program